src/cipher.rs

```diff
@@ -100,6 +100,18 @@ pub extern "C" fn rustls_supported_ciphersuite_get_name(
     }
 }
 
+/// Returns the IANA registered protocol version identifier of the ciphersuite.
+///
+/// See also `RUSTLS_ALL_VERSIONS`.
+#[no_mangle]
+pub extern "C" fn rustls_supported_ciphersuite_protocol(
+    supported_ciphersuite: *const rustls_supported_ciphersuite,
+) -> u16 {
```

Comment:
Return `rustls_tls_version` perhaps?

Reply:
Ah! I forgot we had that. Good catch, thank you

Reply:
Switched to use this enum. I also had to add an `Unknown` (0x0000) variant to make things play nice with the `ffi_panic_boundary!` code.